internal/processor/printer.go

```diff
@@ -105,6 +105,12 @@ func (p *SDCPrinter) ReProcess(pfr *pkg.PrintFileRequest) error {
 	// increment the number of attempts
 	numberOfAttempts := pfr.Attempts
 	pfr.Attempts = numberOfAttempts + 1
+	// log an error for each retry
+	if pfr.Attempts > 1 {
```

Comment:
this will always be more than 1 so we can just remove this if statement
https://github.com/ONSdigital/ras-rm-print-file/blob/234d313d04a8cc1ad3a2e98032fe20a7382ffea8/internal/database/datastore.go#L81